### ISSUE TITLE
paypal cc payment wants post (zip) code with no spaces

### ISSUE THREAD

**Issue by brittainmark** [lat9/paypalr#44]
PayPal does not like spaces in post (zip) codes SN4 0NW not accepted SN40NW accepted.

This is not a big deal, you just have to remove the space in the form.

I believe that this only affects the credit card form.

**Comment by lat9** [comment]
Yep, similar to [this](https://github.com/lat9/paypalr/issues/14) issue, but in a different spot in the payment request.
